tests/by-util/test_unexpand.rs

```diff
@@ -235,3 +235,35 @@ fn test_tabs_shortcut_with_too_large_size() {
 
     new_ucmd!().arg(arg).fails().stderr_contains(expected_error);
 }
+
+#[test]
+fn test_is_directory() {
+    let (at, mut ucmd) = at_and_ucmd!();
+    let dir_name = "dir";
+    at.mkdir(dir_name);
+
+    ucmd.arg(dir_name)
+        .fails()
+        .stderr_contains(format!("unexpand: {}: Is a directory", dir_name));
+}
+
+#[test]
+fn test_multiple_files() {
+    let (at, mut ucmd) = at_and_ucmd!();
+
+    at.write("file", "content");
+    at.write("file1", "a        b");
+
+    ucmd.args(&["file", "file1"])
+        .succeeds()
+        .stdout_is("contenta        b");
+}
+
+#[test]
+fn test_one_nonexisting_file() {
+    let (at, mut ucmd) = at_and_ucmd!();
+
+    ucmd.arg("asdf.txt")
```

Comment:
The `at` causes an "unused variable"-warning. Instead of `at_and_ucmd!()` you can simply use `new_cmd!()` in this case:
```suggestion
    new_cmd!()
        .arg("asdf.txt")
```